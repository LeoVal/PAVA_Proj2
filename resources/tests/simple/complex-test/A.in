#include ".\resources\tests\simple\complex-test\A.include"

public void print() {
	var someValue = new includedAlias();
	System.out.println(#"will #{includedAlias.staticFunction(getIncluded())} and #{includedAlias.staticFunction(someValue)} be the same?");
}