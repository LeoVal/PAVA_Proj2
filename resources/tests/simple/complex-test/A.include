#include ".\resources\tests\simple\complex-test\A.include2"
#getset private includedAlias included; { get; set; }