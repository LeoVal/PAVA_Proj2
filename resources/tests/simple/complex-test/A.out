
private someIncludedType included;

private someIncludedType getIncluded(){
	 return this.included;
}

private void setIncluded(someIncludedType included){
	 this.included=included;
}



public void print() {
	someIncludedType someValue = new someIncludedType();
	System.out.println("will " + (someIncludedType.staticFunction(getIncluded())) + " and " + (someIncludedType.staticFunction(someValue)) + " be the same?");
}